exercises/isogram/canonical-data.json

```diff
@@ -124,7 +122,9 @@
     {
       "uuid": "0d0b8644-0a1e-4a31-a432-2b3ee270d847",
       "description": "word with duplicated character and with two hyphens",
-      "comments": ["This test aims to catch buggy implementations that check for duplicate spaces or hyphens."],
+      "comments": [
+        "This test aims to catch buggy implementations that check for duplicate spaces or hyphens."
+      ],
```

Comment:
in contrast to a different place where a long comment was chosen to be on the same line as the `[` and `]`, I see that here it was chosen to be on different lines than `[` and `]`. I don't have an explanation, but I don't need one.